.github/workflows/main.yaml

```diff
@@ -124,3 +130,9 @@ jobs:
       run: |
         pre-commit install
         makim tests.lint
+
```

Comment:
You don't need these changes here. Just keep the implementation in the job above, please. Thanks!